src/process/process.rs

```diff
@@ -63,8 +63,9 @@ pub struct Callback {
     pub pc: usize
 }
 
-#[repr(C,packed)]
```

Comment:
I'm not sure what the rust equivalent would be, but one habit from C that helps verify no funny padding business is going on:

    _Static_assert(sizeof(struct type_t) == 32)